acceptance-tests/tests/build.gradle

```diff
@@ -14,6 +14,13 @@
 plugins {
   id 'org.web3j' version '4.11.1'
   id 'org.web3j.solidity' version '0.4.0'
+  id 'maven-publish'
+}
+
+publishing {
+  publications {
+    mavenJava(MavenPublication) { artifactId 'acceptance-tests-tests' }
+  }
```

Comment:
would this result in publishing these artifacts to artifactory?  We currently publish libs to hyperledger jfrog as part of the release using artifactoryPublish task

Reply:
That's a good question. I've got to admit added this to avoid a gradle build failure and wasn't quite sure of the side-affects (other than it fixed the build! :) )

Reply:
OK - think the latest commit sorts this out @siladu. I've removed the publish specs and just disabled `jar` building for the new `shanghai` project.